pages/code-editor/controllable-height.page.tsx

```diff
@@ -15,6 +15,9 @@ export default function () {
   useEffect(() => {
     import('ace-builds').then(ace => {
       ace.config.set('basePath', './ace/');
+      ace.config.set('themePath', './ace/');
+      ace.config.set('modePath', './ace/');
+      ace.config.set('workerPath', './ace/');
```

Comment:
is a dry run to live necessary here to make sure that there isn't customers who depends on this?

Reply:
Approved, feel free to merge after running the dry run